membership-attribute-service/app/actions/CommonActions.scala

```diff
@@ -52,10 +53,18 @@ class AuthenticatedUserAndBackendRequest[A](
     val user: UserFromToken,
     val touchpoint: TouchpointComponents,
     val request: Request[A],
-) extends WrappedRequest[A](request)
+) extends WrappedRequest[A](request) {
+  implicit val logPrefix: LogPrefix = new LogPrefix {
+    override def message: String = user.identityId
+  }
+}
 
 class AuthAndBackendRequest[A](
     val redirectAdvice: RedirectAdviceResponse,
     val touchpoint: TouchpointComponents,
     request: Request[A],
-) extends WrappedRequest[A](request)
+) extends WrappedRequest[A](request) {
+  implicit val logPrefix: LogPrefix = new LogPrefix {
+    override def message: String = redirectAdvice.userId.getOrElse("no-identity-id")
+  }
```

Comment:
whatever kind of incoming request we get, we try to provide a suitable implicit log prefix.
we can later add things here if we need more information.